chunky/src/java/se/llbit/chunky/renderer/DefaultRenderManager.java

```diff
@@ -323,6 +325,10 @@ public void run() {
     }
   }
 
+  protected void startRender(Renderer renderer) throws InterruptedException {
+    renderer.render(this);
+  }
```

Comment:
Is this for plugins?

Reply:
I believe I added this to be overridable by custom RenderManagers to add pre-start logic.

Reply:
e.g. setting data in the new worker state / ray factories